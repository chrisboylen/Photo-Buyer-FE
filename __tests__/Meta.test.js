import { mount } from "enzyme";
import wait from "waait";
import toJSON from "enzyme-to-json";
import { MockedProvider } from "react-apollo/test-utils";
import Meta from "../components/Meta";

describe("META", () => {
  it("renders and matches snapshot", async () => {
    const wrapper = mount(
      <MockedProvider>
        <Meta />
      </MockedProvider>
    );
    await wait();
    wrapper.update();

    expect(toJSON(wrapper.find("Meta"))).toMatchSnapshot();
  });
});
