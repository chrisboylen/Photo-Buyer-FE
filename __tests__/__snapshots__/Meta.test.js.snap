// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`META renders and matches snapshot 1`] = `
<Meta>
  <SideEffect(Head)>
    <Head />
  </SideEffect(Head)>
</Meta>
`;
